kube/services/revproxy/gen3.nginx.conf/guppy-service.conf

```diff
@@ -3,4 +3,5 @@
               set $upstream_guppy http://guppy-service.$namespace.svc.cluster.local;
               rewrite ^/guppy/(.*) /$1 break;
               proxy_pass $upstream_guppy;
+              client_max_body_size 0;
```

Comment:
Is it really necessary to disable client request body size?

Reply:
It's for asking manifest entries (e.g. get `file_id` with a lot of `subject_id`s in filters), the request payload will be big